tensorflow/inference/docker/2.3.0/py37/Dockerfile.eia

```diff
@@ -1,10 +1,16 @@
 FROM ubuntu:18.04
 
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get autoremove -y \
```

Comment:
@jinpengqi why is this autoremove line added here? Can we remove it?

Reply:
Resolved.